code/game/machinery/doors/multi_tile.dm

```diff
@@ -17,4 +19,23 @@
 	icon = 'icons/obj/doors/Door2x1glass.dmi'
 	opacity = 0
 	glass = 1
-	assembly_type = /obj/structure/door_assembly/multi_tile
\ No newline at end of file
+	assembly_type = /obj/structure/door_assembly/multi_tile
+
+/obj/machinery/door/airlock/multi_tile/setup_hatch()
+
+	if (overlays != null)
```

Comment:
I'm curious, is this check necessary/what is it for? I imagine the `..()` call for `New()` initializes the object to already have `overlays = list()`.